src/ctapipe/resources/stats_calc_config.yaml

```diff
@@ -0,0 +1,37 @@
+StatisticsCalculatorTool:
+  allowed_tels: [1,2,3,4]
+  dl1a_column_name: "image"
+  output_column_name: "statistics"
+
+PixelStatisticsCalculator:
+  stats_aggregator_type: [["type", "*", "SigmaClippingAggregator"]]
+  chunk_shift: 1000
+  faulty_pixels_fraction: 0.1
+  outlier_detector_list: [
```

Comment:
Why use the json syntax here? I think using the yaml syntax is much more readable:

```
outlier_detector_list:
   -  name: ...
      apply_to: median
      config: 
       ...
   -  name: ...
      apply_to: median
      config: 
       ...
```